costmodel/costmodel.go

```diff
@@ -974,7 +968,7 @@ func GetPVCost(pv *costAnalyzerCloud.PV, kpv *v1.PersistentVolume, cp costAnalyz
 	return nil
 }
 
-func getNodeCost(cache ClusterCache, cp costAnalyzerCloud.Provider) (map[string]*costAnalyzerCloud.Node, error) {
+func getNodeCost(cache clustercache.ClusterCache, cp costAnalyzerCloud.Provider) (map[string]*costAnalyzerCloud.Node, error) {
```

Comment:
Perhaps we should refactor this entire method into provider as something like: 
```go
func GetNodeCost(cp Provider) (map[string]*Node, error)
```

It does appear to create an "instanced" version of provider `Node`s. Not sure of the best approach here with regards to the `ApplyReservedInstancePricing` method call below.